seed/seed20.go

```diff
@@ -291,7 +291,7 @@ func (s *seed20) Copy(seedDir string, opts CopyOptions, tm timings.Measurer) (er
 	}
 
 	// these files are the files we can safely copy without an modifications
```

Comment:
Maybe having a comment on why grubenv is not copied and how it is created elsewhere would be good